src/Platform/OpenGL/Shaders/god_ray_fragment.glsl

```diff
@@ -0,0 +1,57 @@
+#include "Library/directional_light.glsl"
+
+in vec2 TexCoord;
+out vec4 OutColor;
+
+uniform sampler2D albedoTex;
+uniform sampler2D normalTex;
+uniform sampler2D materialTex;
+uniform sampler2D depthTex;
+
+struct Camera
```

Comment:
my failure, I see camera is redaclared in each shader. For this PR is fine, but later I need to move this to a Library utils file. Just a not for myself